inventory/group_vars/manager/00_main.yml

```diff
@@ -32,8 +32,9 @@ chrony_servers:
   - time.cloudflare.com
 
 tinyproxy_allowed_networks:
-  - 10.10.21.0/24
-  - 10.10.23.0/24
+  - 10.0.0.0/8
+  - 172.16.0.0/12
+  - 192.168.0.0/16
```

Comment:
What is this network used for or where does the network need access to the proxy? If we use the network explicitly, would it probably be good to enter the network in the overview of networks (documentatio/System_Networks.md) and describe it there?

Reply:
> What is this network used for or where does the network need access to the proxy? 

Netbox wants to reach the internet because it uses GitHub as a data source.

Overall, you are right that it is not necessary to allow all the above networks. It is better to set up Docker to use the selected subnet. 

As a quick fix, I whitelisted just 172.16.0.0/12, as I believe it is the one docker used (by default). 
Also, I mentioned it in the docs. c4ab497eaece42cad38039bc533c3a5738175cec